apps/web/src/app/github/page.tsx

```diff
@@ -136,21 +104,37 @@ export default function GitHubPage() {
         </div>
       )}
 
-      <div className="rounded-lg bg-white p-6 shadow-md">
+      <div className="rounded-lg bg-white p-6 shadow-md dark:bg-gray-900">
         <h2 className="mb-4 text-xl font-semibold">Manage GitHub App</h2>
-        <p className="mb-4">
+        <p className="mb-4 text-gray-600 dark:text-gray-400">
           You can manage your GitHub App installation, including adding or
           removing repositories, through GitHub.
         </p>
         <a
           href="https://github.com/settings/installations"
           target="_blank"
           rel="noopener noreferrer"
-          className="inline-block rounded bg-gray-200 px-4 py-2 text-gray-800 hover:bg-gray-300"
+          className="inline-block rounded bg-gray-200 px-4 py-2 text-gray-800 hover:bg-gray-300 dark:bg-gray-700 dark:text-gray-200 dark:hover:bg-gray-600"
         >
           Manage on GitHub
         </a>
       </div>
     </div>
   );
 }
+
+function GitHubPageWithProvider() {
+  return (
+    <GitHubAppProvider>
+      <GitHubPageContent />
+    </GitHubAppProvider>
+  );
+}
+
+export default function GitHubPage() {
+  return (
+    <Suspense fallback={<GithubPageLoading />}>
+      <GitHubPageWithProvider />
+    </Suspense>
+  );
+}
```

Comment:
lets just merge this into one func. doesnt need to be two
```suggestion
export default function GitHubPage() {
  return (
    <Suspense fallback={<GithubPageLoading />}>
      <GitHubAppProvider>
        <GitHubPageContent />
      </GitHubAppProvider>
    </Suspense>
  );
}
```